### PR TITLE
chore: updated supported language list and language utility

### PR DESCRIPTION
**Pull request by likhith-deriv**
Updated supported language list to match with Deriv-app and added functionality to filter languages.
Also removed `exports` prop from package.json to bring compatibility with TS 4.6
Included `react/jsx-runtime` in the build of the library to support old App using React versions

### CODE REVIEW

**Review by matin-deriv — APPROVED** [review]

File: src/utils/lang-utils.ts [hunk 1]
```diff
@@ -34,3 +34,22 @@ export const loadIncontextTranslation = (
     document.head.appendChild(jipt);
   }
 };
+
+/**
+ * Filter out unsupported languages and return an Object containing language code and language name
+ * @param excludedLanguages
+ * @returns Object containing language code and language name
+ */
+export const getAllowedLanguages = (
```
Comment (amina-deriv) [inline src/utils/lang-utils.ts:43]: Ahould we specify the platform as a prop to "getAllowedLanguages" function?

**Review by suisin-deriv — APPROVED** [review]

File: src/utils/lang-utils.ts (hunk 1 above)
Comment (likhith-deriv) [reply src/utils/lang-utils.ts:43]: I think the platform can handle that inside the Platform codebase. We can have same  function called with separate params for different environments as well. This was library function will be pure

**Review by amina-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Review by matin-deriv — APPROVED** [review]

File: src/utils/lang-utils.ts [hunk 2]
```diff
@@ -34,3 +34,22 @@ export const loadIncontextTranslation = (
...
+  excludedLanguages: Omit<LanguageCode, "ACH">[] = []
```
Comment (ali-hosseini-deriv) [inline src/utils/lang-utils.ts:44]: where does this hardcoded ACH come from?

**Review by ali-hosseini-deriv — APPROVED** [review]

File: src/utils/lang-utils.ts (hunk 2 above)
Comment (likhith-deriv) [reply src/utils/lang-utils.ts:44]: Its a key present in [src/utils/constants.ts](https://github.com/deriv-com/translations/pull/41/files#diff-e8d0a358ae2fb090c56f00a058050983cc9691b7d3622ddd443170017123e7d9)

**Pull request merged**